x-pack/plugin/transform/src/internalClusterTest/java/org/elasticsearch/xpack/transform/checkpoint/TransformGetCheckpointIT.java

```diff
@@ -98,25 +101,28 @@ public void testGetCheckpoint() throws Exception {
                 .filter(i -> i.getShardRouting().primary())
                 .sorted(Comparator.comparingInt(value -> value.getShardRouting().id()))
                 .mapToLong(s -> s.getSeqNoStats().getGlobalCheckpoint())
+                .filter(Objects::nonNull)
                 .sum(),
             checkpointSum
         );
+        deleteIndices(indices);
```

Comment:
If the test fails will the `deleteIndices` still be called? Would it make sense to move these to an `After` method. I guess not all tests need to delete the indices, but maybe we could just ignore a failure on a delete attempt in that situation and always try to delete the indices.